README.md

```diff
@@ -84,8 +85,9 @@ The resulting binaries can be found in `build/bin/<Debug_x86|Release_x86>`.
 
 ### Linux
 
-When building on Linux you need to have g++ installed.
-Since you are most likely on a 64bit machine you will also need multilib for compiling for 32bit.
+When building on Linux you need to have g++ 13 or later installed.
+Since you are most likely on a 64-bit machine you will also need multilib for compiling for 32-bit.
+Using other compilers may also be possible, but they are not guaranteed to work out of the box. You may need to edit [premake5.lua](premake5.lua) to ensure compatibility with other compilers such as Clang.
```

Comment:
I think i'd just leave away the last sentence because i'd not recommend using a compiler that is not supported without making custom changes.
If someone feels like making Clang support it could still be added to the repo